internal/cli/fmt.go

```diff
@@ -0,0 +1,142 @@
+package cli
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/posener/complete"
+
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	configpkg "github.com/hashicorp/waypoint/internal/config"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+)
+
+type FmtCommand struct {
+	*baseCommand
+
+	flagWrite bool
+}
+
+func (c *FmtCommand) Run(args []string) int {
+	// Initialize. If we fail, we just exit since Init handles the UI.
+	if err := c.Init(
+		WithArgs(args),
+		WithFlags(c.Flags()),
+		WithNoConfig(),
+		WithClient(false),
+	); err != nil {
+		return 1
+	}
+
+	// If we have too many args, error immediately.
+	if len(c.args) > 1 {
+		c.ui.Output("At most one argument is expected.\n\n"+c.Help(), terminal.WithErrorStyle())
```

Comment:
I noticed this should maaybe be increased if FILE is provided? So if you want to format a waypoint.hcl in a different  folder but to STDOUT with `-write=false`:

```
brian@localghost:docker % waypoint fmt angular/waypoint.hcl -write=false                                                                                                                                                                ±[●][main]
! At most one argument is expected.
```

Reply:
Don't think this is a bug, flags in Go have to be before args, so `-write=false angular/waypoint.hcl` should work.